netplay/joinroom.cpp

```diff
@@ -25,9 +25,10 @@ JoinRoom::JoinRoom(QWidget *parent)
     playerName->setPlaceholderText("Player Name");
     layout->addWidget(playerName, 0, 0);
 
-    QLabel *serverLabel = new QLabel("Server", this);
-    serverLabel->setAlignment(Qt::AlignRight | Qt::AlignVCenter);
-    layout->addWidget(serverLabel, 0, 1);
+    inputDelay = new QLineEdit(this);
```

Comment:
Same comment here about the input validator to ensure only positive integers are entered